spec/sample-data/valid/control-statement-04.json

```diff
@@ -0,0 +1,80 @@
+{
+	"subject_entity": {
```

Comment:
We don't check this (yet), but all(?) the rest of these examples use two-space indent, not tabs...

Reply:
I was wrong - we _do_ check it! https://github.com/openc/openc-schema/pull/66